integrations/drift/drift.go

```diff
@@ -0,0 +1,115 @@
+package drift
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/jipiboily/forwardlytics/integrations"
+)
+
+// Drift integration
+type Drift struct {
+	api service
+}
+
+type service interface {
+	request(string, string, []byte) error
+}
+
+type driftAPIProduction struct {
+	baseUrl string
+}
+
+type apiSubscriber struct {
+	Attributes map[string]interface{} `json:"attributes"`
+	CreatedAt  int64                  `json:"createdAt"`
+	UserId     string                 `json:"userId"`
+	OrgId      string                 `json:"orgId"`
+}
+
+type apiEvent struct {
+	OrgId      string                 `json:"orgId"`
+	UserId     string                 `json:"userId"`
+	Event      string                 `json:"event"`
+	CreatedAt  int64                  `json:"createdAt"`
+	Attributes map[string]interface{} `json:"attributes"`
+}
+
+// Identify forwards and identify call to Drift
+func (d Drift) Identify(identification integrations.Identification) (err error) {
+	s := apiSubscriber{}
+	s.UserId = string(identification.UserID)
+	s.CreatedAt = identification.Timestamp
+	s.OrgId = orgID()
+	// Add custom attributes
+	s.Attributes = identification.UserTraits
+	s.Attributes["forwardlyticsReceivedAt"] = identification.ReceivedAt
+	s.Attributes["forwardlyticsTimestamp"] = identification.Timestamp
+	payload, err := json.Marshal(s)
+
+	err = d.api.request("POST", "identify", payload)
```

Comment:
Sure. More logging is a good idea in general. It's a bit hard to know what went wrong per now because of the logging that is a bit minimal. I'll add it everywhere you mentioned as a start.